### ISSUE TITLE
Получение данных

### ISSUE THREAD

**Issue by viktorovich87** [asd95/tt-adminui#6]
Получение данных через индекс в массиве - это не совсем корректный способ. 
в данном примере/проекте с этим кодом - логика будет работать, но если  поменяются входящие данные - всё поломается..

**Comment by asd95** [comment]
Да, знал что заметите. Я подстроился под компонент связанный с таблицей. Так как он работает с двумя таблицами, я сделал вначале страницу Suport. Дальше когда перешел в Chart данные которые получал не со всем совпадали с тем что должна получать таблица. Глупо. Нужно было просто переделать сам компонент отвечающий за таблицу. Ну а данные лучше получать в объекте из-за лучшей доступности. 
Вывод: структуру данных поменять, изменить компонент табилцы. Спасибо)

**Comment by viktorovich87** [comment]
надо стараться не подстраиваться по входящие данные, а стараться написать код таким способом, чтоб при любом изменении входящих данных показывалось что-то.. либо стояла проверка, мол данных нет - пустая таблица.. потому как в реальном проекте формат данных может поменяться, либо не приходить вообще и такой код как `arr[0]` либо `arr.objItem` может отсутствовать и будет креш приложения.. и в этот момент исправить код моментально не получается, поэтому надо пробовать писать код, который не стрельнет ошибку
